tests/errors.spec.js

```diff
@@ -0,0 +1,34 @@
+var m = require('mochainon')
```

Comment:
I'm going to enable both TS and JS tests in the karma config and convert this one to TS then